tests/components/plugwise/test_init.py

```diff
@@ -257,7 +257,8 @@ async def test_update_device(
     # Add a 2nd Tom/Floor
     data.devices.update(TOM)
     with patch(HA_PLUGWISE_SMILE_ASYNC_UPDATE, return_value=data):
-        async_fire_time_changed(hass, utcnow + timedelta(minutes=2))
+        freezer.tick(timedelta(minutes=10))
```

Comment:
minutes=1 should work, it's the default interval.

Reply:
Just down streaming :)

Reply:
![image](https://github.com/user-attachments/assets/dc856d32-d064-4c3f-87b4-05ca7405947c)

Reply:
Upstream staat dit er.

Reply:
Fixed